.github/workflows/release-helm-chart.yaml

```diff
@@ -0,0 +1,56 @@
+name: Release Helm Chart
+
+on:
+  pull_request:
+    types:
+      - closed
+
```

Comment:
This workflow file looks great. But we actually only want to release on tags. Also, as I mentioned in a comment, I recommend pulling the work into a separate PR.